### PR TITLE
Vue password keeper

### PR DESCRIPTION
**Pull request by coderonfleek**

### CODE REVIEW

File: articles/2020-01-16-title.markdown [hunk 1]
```diff
@@ -0,0 +1,752 @@
...
+
+After your selection, the Vue CLI begins scaffolding your new Vuejs application.
+
+## Creating an Auth0 Passwordless Authentication Plugin
+
+To work with Passwordless, you will need the [Auth0 Lock](https://github.com/auth0/lock/) NPM package. This package contains all you need for authenticating your application with Passwordless.
+
+To use this package effectively in the Vuejs application just scaffolded, a Vuejs authentication plugin will be created and exported to be used within the application. This plugin will make use of the Auth0 Lock package to set up authentication functions and parameters that can be made use of in the application pages and components.
+
+Install the Auth0 Lock package by running the command:
```
Comment (SarahJay55) [inline articles/2020-01-16-title.markdown]: We need to tell them to `cd` into the project first

File: articles/2020-01-16-title.markdown [hunk 2]
```diff
@@ -0,0 +1,752 @@
...
+
+```
+npm install auth0-lock
+```
+
+**NB**: If after installing this package or at any point in this exercise you encounter an error regarding `core-js`, simply run `npm install core-js` to fix the issue.
```
Comment (SarahJay55) [inline articles/2020-01-16-title.markdown]: Let's have this say **Note**: instead

File: articles/2020-01-16-title.markdown [hunk 3]
```diff
@@ -0,0 +1,752 @@
...
+
+Within the `src` folder, create a new folder named **auth** and within this folder, create a file named `index.js` and place the following code inside the file:
+
+```javascript
```
Comment (SarahJay55) [inline articles/2020-01-16-title.markdown]: we need to add the file path at the top of each code block (commented out)

File: articles/2020-01-16-title.markdown [hunk 4]
```diff
@@ -0,0 +1,752 @@
...
+
+Install the `crypto-js` package by running the following command:
+
+```
+npm install crypto-js
+```
+
+All that is needed for the application is now in place. Go into the `App.vue` file and replace everything inside it with the code below:
+
+```javascript
```
Comment (SarahJay55) [inline articles/2020-01-16-title.markdown]: please remember, let's add the file path commented out to the top of each code block

File: articles/2020-01-16-title.markdown [hunk 5]
```diff
@@ -0,0 +1,752 @@
...
+
+Time to take the application for a spin. Serve the application in your browser by running the following command:
+
+```
+npm run serve
+```
+
+If all instructions have been properly followed you should see the screen below at the address `http://localhost:8080`:
+
+**NB**: As stated earlier in the article, if you run into any error regarding `core-js`, simply run the following command to install the package:
```
Comment (SarahJay55) [inline articles/2020-01-16-title.markdown]: Change this to **Note**:

**Comment by kapehe-ok** [comment]
this application was changed. closed.

**Pull request closed without merging**